<!---->

  
<launch>  
    <node name="kittiHelper" pkg="aloam_velodyne" type="kittiHelper" output="screen"> 
        <param name="dataset_folder" type="string" value="/media/jxl/0C3B04470C3B0447/KITTI_odometry/" />
        <param name="sequence_number" type="string" value="00" />
        <param name="to_bag" type="bool" value="false" />
        <param name="output_bag_file" type="string" value="/media/jxl/0C3B04470C3B0447/KITTI_odometry/kitti_00.bag" />
        <param name="publish_delay" type="int" value="1" />
        <!--remap from="/velodyne_points" to="/helper_velodyne_points" /-->
    </node>
</launch>
